pubpub/pubpub
# Export fixes

This PR bundles a couple of export fixes.

- Adds license information to all exports (Resolves #737)
- Fixes font sizes for `ol` and `ul` with an override (Resolves #815)
- So that style updates to Pubs carry over automatically to PDF and HTML exports, builds the css for pagedjs directly from the editor styles
- Bumps the pandoc version to the latest (2.9.2.1) to support different jats export targets
- Includes pandoc's templates in the pubpub directory so we can use them to generate various files
- Specifies a template file for most pandoc export types we support, which allows us to create more structured exports
- Specifies pandoc yaml metadata based on the needs of common pandoc templates, with a special version for jats

_Test Plan_
1. Visit a complex pub with many authors, ie: http://hdsr.mitpress.mit.edu/pub/ct67j043/draft
1. Download the pre-generated PDF that already exists
1. Go to draft, make and then undo a change, and export each file type
1. For PDF and HTML, verify that ul and li sizes are the same as main text, that the license text appears, and that otherwise, the PDF matches the previous download.
1. For other types, verify that they include common metadata, including date published, authors, license, title, collection, etc.
1. Create a pub. Include some shadow authors with affiliations, as well as yourself with affiliations and export the PDF and JATS.
1. Verify that shadow authors and affiliations show up correctly, and that your affiliation shows up correctly.
